src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -10,11 +10,16 @@ import {
   __InputStylesNames,
   ElementProps,
 } from '@mantine/core';
+import dayjs from 'dayjs';
```

Comment:
dayjs import must be before React import

Reply:
updated